src/traces/image/plot.js

```diff
@@ -13,14 +13,24 @@ var Lib = require('../../lib');
 var xmlnsNamespaces = require('../../constants/xmlns_namespaces');
 var constants = require('./constants');
 
+function compatibleAxis(ax) {
+    return ax.type === 'linear' &&
+        // y axis must be reversed but x axis mustn't be
+        ((ax.range[1] > ax.range[0]) === (ax._id.charAt(0) === 'x'));
+}
+
 module.exports = function plot(gd, plotinfo, cdimage, imageLayer) {
     var xa = plotinfo.xaxis;
     var ya = plotinfo.yaxis;
 
+    var supportsPixelatedImage = !Lib.isSafari() && !gd._context._exportedPlot;
```

Comment:
Safari may actually be supported. I need to test all major browsers and change this line accordingly.

In the case of an exported figure, we can't know whether a given SVG renderer implements `image-rendering: pixelated` so we shouldn't rely on it.

Reply:
> Safari may actually be supported.

It's not. iOS is also a no-go as well as IE.